### PR TITLE
[HID Bravura] Converting dashboards to lens

### PR DESCRIPTION
**Pull request by P1llus**
## What does this PR do?

Bumps format_version to 2.7.0, min version to 8.7.1 and converts all non-lens dashboards to lens.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

hid-event-analysis
hid-log-issues-summary
hid-administrative-disabled-profiles

### CODE REVIEW

File: packages/hid_bravura_monitor/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.9.1"
```
Comment (bhapas) [inline packages/hid_bravura_monitor/changelog.yml:2]: 1.9.0 ?

**Review by bhapas — COMMENTED** [review]
Possible to add some screenshots?

LGTM otherwise

**Comment by P1llus** [comment]
> Possible to add some screenshots?
> 
> LGTM otherwise

I can add a few, but there is not that much test data, and it has a very large amount of dashboards :) The layout did not really change, even the visualization type (like bar to bar) was kept the same)

**Comment by bhapas** [comment]
> > Possible to add some screenshots?
> > LGTM otherwise
> 
> I can add a few, but there is not that much test data, and it has a very large amount of dashboards :) The layout did not really change, even the visualization type (like bar to bar) was kept the same)

Not really necessary if the visualization layout did not change though.

**Review by bhapas — APPROVED** [review]

**Pull request merged**